lib/presentation/profile/profile_screen.dart

```diff
@@ -287,6 +283,17 @@ class _UserProfilePageState extends State<UserProfilePage> {
                                 ),
                               ),
                             ],
+                          ] else ...[
+                            // TODO only for MVP (remove later)
+                            const SizedBox(height: 40),
+                            PillButton(
+                              text: 'Create account or sign in',
+                              onTap: () async {
+                                await context.router.push(const AuthRoute());
+                                // Refresh profile
+                                profileBloc.add(GetUserProfile());
```

Comment:
```suggestion
                                BlocProvider.of<ProfileBloc>(context).add(GetUserProfile());
```
or 
```suggestion
                                context.read<ProfileBloc>().add(GetUserProfile());
```

Reply:
Build context should not be used across `async`!